### DISCUSSION TITLE
error on App.js

### DISCUSSION THREAD

**Discussion by jayomayan** [filamentphp/filament#6416] [Help]
has anyone encounter this error?
the profile menu dropdown not working, the sidebar collapse if enabled it will just show white black screen.

`app.js?id=e86eecc2b1e06821fc6e52592bc106bd:180 Uncaught SyntaxError: Unexpected token '}' (at app.js?id=e86eecc2b1e06821fc6e52592bc106bd:180:141512)`

**Comment by danharrin** [comment]
I can't reproduce this. What plugins do you have installed?

**Reply by redcrach** [reply]
I have a similar error! Just installed a new Laravel all with sail and added filament, tried to do a simple resource that uses `FileUpload:class` but I got this js issue... and it seems all js related stuff is broken (dropdowns, datepicker...)

**Reply by danharrin** [reply]
Please help me reproduce it so I can fix it...

**Comment by danharrin** [comment]
It's fixed in the latest version @jayomayan @redcrach, please clear your browser cache too
